app/Http/Controllers/Api/v1/User/Controllers/GenerateTokenController.php

```diff
@@ -42,13 +44,42 @@ public function __invoke(Request $request): JsonResponse
         try {
             $user = $this->userLookup->getUpdatedUser($user, $azureUser->principal_name);
         } catch (ClientExceptionInterface) {
+            $log = new Log();
+            $log->user_id = $user->id;
+            $log->severity = '5';
```

Comment:
Why is 5 a string? x)

Reply:
during migration, enum can only be a string

Reply:
Then let this be dictated by an enum

(Doesn't have to be an enum btw. Just create a Severity class with static string constants)

Reply:
`Severity.INFO` klinkt bijvoorbeeld veel chiller dan `6` :)

Reply:
```php
class Severity {
    ...
    public static $INFO = 6;
    public static $DEBUG = 7;
}
```